Button.js

```diff
@@ -33,20 +34,20 @@ const styles = StyleSheet.create({
 
 export default class Button extends Component {
     static propTypes = {
-        backgroundColor: React.PropTypes.string,
-        children: React.PropTypes.oneOfType([
-            React.PropTypes.arrayOf(React.PropTypes.node),
-            React.PropTypes.node,
+        backgroundColor: PropTypes.string,
+        children: PropTypes.oneOfType([
+            PropTypes.arrayOf(PropTypes.node),
+            PropTypes.node,
         ]),
-        containerStyle: View.propTypes.style,
-        elevation: React.PropTypes.number,
-        onPress: React.PropTypes.func.isRequired,
-        text: React.PropTypes.string,
-        textContainerStyle: View.propTypes.style,
-        textStyle: Text.propTypes.style,
-        touchableContainerStyle: View.propTypes.style,
-        underlayColor: React.PropTypes.string,
-        uppercase: React.PropTypes.bool,
+        containerStyle: PropTypes.style,
```

Comment:
Please use `ViewPropTypes` instead of `PropTypes` (https://github.com/facebook/react-native/commit/53905a537a58d028fc5eb1c4e9a8ec967d8cd396)